html-links/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Links</title>
+</head>
+<body>
+  <ul>
```

Comment:
Woohoo! Great job! Just one small request: I think we are missing heading for this page!